### ISSUE TITLE
Support deepin OS?

### ISSUE THREAD

**Issue by ShixiangWang** [r-lib/rig#111]
Deepin is a re-distro based on Debian, what works for Debian shall work for Deepin. Could rig support this OS?

```
$ rig add devel
[INFO] Running `sudo` for adding new R versions. This might need your password.
[ERROR] Unsupported distro: Deepin 20.6, only Ubuntu 18.04, 20.04, 22.04 and Debian 9-11 are supported currently
```

**Comment by gaborcsardi** [comment]
We can probably do that. What's the output of this?
```
cat /etc/os-release
```

**Comment by ShixiangWang** [comment]
```
$ cat /etc/os-release
PRETTY_NAME="Deepin 20.6"
NAME="Deepin"
VERSION_ID="20.6"
VERSION="20.6"
VERSION_CODENAME="apricot"
ID=Deepin
HOME_URL="https://www.deepin.org/"
BUG_REPORT_URL="https://bbs.deepin.org/"
```

**Comment by gaborcsardi** [comment]
Thanks! Can you also include the output of this?
```
cat /etc/debian_version
```

**Comment by ShixiangWang** [comment]
```
$ cat /etc/debian_version
10.10
```

**Comment by gaborcsardi** [comment]
Closed by 8c08328

**Comment by ShixiangWang** [comment]
Thanks
